frontend/src/components/maps/leaflet/InfoSlideOut/BuildingAttributes.tsx

```diff
@@ -27,21 +28,25 @@ export const BuildingAttributes: React.FC<IBuildingAttributes> = ({ buildingInfo
     formatAssessed = '';
   }
 
+  const newLength = buildingInfo.parcels?.length > 3 ? 3 : buildingInfo.parcels?.length;
+  const parcelsCopy = buildingInfo.parcels?.slice(0, newLength);
+
```

Comment:
I know most buildings won't have more than 1 parcel, but i thought I'd do the same as associated buildings just in case